build/release/README.md

```diff
@@ -53,15 +51,47 @@ in the next section with the updated version.
 
 **IMPORTANT** Before tagging the release, open a new PR to update the documentation and example manifest tags to the release version.
 
-To publish a new release build, run the `Tag` action in GitHub actions. Be sure to use the correct branch for the
-release. New minor releases will always be run from master, and patch releases will come from a
-previously created release branch.
+To publish a new patch release build, follow these steps:
+
+1. Make sure all needed PRs are merged to the release branch
+2. Check that integration tests are green (except intermittent issues)
+3. Open a PR to update the version, and merge it
+4. Tag the branch:
+
+    ```console
+    # make sure no files are checked out locally, then proceed:
+    git checkout release-1.10
```

Comment:
Let's not include specific branches or versions, otherwise someone will accidentally copy-paste an old version. What about this?
```suggestion
   git checkout <brannch> # e.g. release-1.10
```